letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
We can't modify well-known URI in anyway: https://github.com/ietf-wg-acme/acme/blob/master/draft-ietf-acme-acme.md#http

So we can't know which database must be selected if there are two or more.

If we add a config parameter like 'letsencript_db' in odoo configuration file, then we can populate registry (request.env) for that database in this scenario. What do you think? Then we can use env['letsencrypt'] model again and inheritance is easier too.